graphs/minimum_spanning_tree_prims.py

```diff
@@ -104,7 +104,7 @@ def deleteMinimum(heap, positions):
     return TreeEdges
 
 
-if __name__ == "__main__":
+if __name__ == "__main__":  # pragma: no cover
```

Comment:
Why?

Reply:
to increase coverage and ignore non-relevant coverage reports at the same time